jobrunner/lib/docker.py

```diff
@@ -47,6 +47,13 @@ class DockerDiskSpaceError(Exception):
     pass
 
 
+def docker_labels(job):
+    return [
+        "--label", f"action={job.action}",
+        "--label", f"workspace={job.workspace}",
+    ]
```

Comment:
Semantic pedantry alert: I'm aware I've not written down my mental model of this anywhere, but to me this is the wrong place for a function like this. I think of `lib/docker.py` as a generic "make docker easier to interact" with module, so not one that knows about "jobs" as such, or other application level concerns. So I'd rather be passing the labels in and have the function which generates them somewhere else.

Reply:
Have addressed, good call. I felt icky writing it, just couldn't quite get to how to do it better.